userdocs/profiles.dev/docs/faq.md

```diff
@@ -4,4 +4,9 @@ sidebar_position: 10
 
 # FAQ
 
-coming soon
+Q. **Can I use something other than GitHub to host my GitOps repo?**
+
+A. At the moment, GitHub is the only git provider supported by `pctl`. BUT this only affects
+   the `pctl install` command when using the `--create-pr` flag. If you wish to use, say, GitLab
+   to manage your cluster, you can use `pctl` to generate the profile manifests to a local directory,
+   and then commit them to your Flux-synced repo manually.
```

Comment:
I feel like this should be reworded to: we support all. However if doing create-pr only github is currently supported. The PR-ing is a nice to have (IMO), the rest of the core functionality all works